services/src/main/java/org/fao/geonet/api/ApiUtils.java

```diff
@@ -379,4 +374,9 @@ public static String processRequestValidation(BindingResult bindingResult, Resou
             return "";
         }
     }
+
+    public static ResourceBundle getMessagesResourceBundle(Enumeration<Locale> locales) {
+        Locale locale = languageUtils.parseAcceptLanguage(locales);
```

Comment:
I'm getting 
```

java.lang.NullPointerException: null
	at org.fao.geonet.api.ApiUtils.getMessagesResourceBundle(ApiUtils.java:379) ~[classes/:?]
	at org.fao.geonet.api.records.MetadataInsertDeleteApi.create(MetadataInsertDeleteApi.java:465) 
```

I'm not sure we can inject static languageUtils ?

Reply:
@fxprunayre I've updated the code, should be fine now.